Sources/ProjectSpec/TestPlan.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import JSONUtilities
+
+public struct TestPlan: Equatable {
+    public var path: String
+    public var defaultPlan: Bool
+
+    public init(path: String, defaultPlan: Bool = false) {
+        self.defaultPlan = defaultPlan
+        self.path = path
+    }
+
+    public static func == (lhs: TestPlan, rhs: TestPlan) -> Bool {
+        lhs.path == rhs.path && lhs.defaultPlan == rhs.defaultPlan
+    }
+}
+
+
+extension TestPlan: JSONObjectConvertible {
+
+    public init(jsonDictionary: JSONDictionary) throws {
+        path = try jsonDictionary.json(atKeyPath: "path")
+        defaultPlan = jsonDictionary.json(atKeyPath: "defaultPlan") ?? false
+    }
+}
+
+extension TestPlan: JSONEncodable {
+    public func toJSONValue() -> Any {
+        [
+            "path": path,
+            "defaultPlan": defaultPlan,
```

Comment:
Maybe we could go with `default` because apple is using the term `default` it in the `xscheme`
But on the other hand `reference` is also a worse name than `path`. 
Let's discuss it. :)
```xml
<TestPlans>
    <TestPlanReference
        reference = "container:first.xctestplan"
        default = "YES">
    </TestPlanReference>
    <TestPlanReference
        reference = "container:second.xctestplan">
    </TestPlanReference>
</TestPlans>
```

Reply:
`default` is a reserved word and should not be used as a variable name or similar. Even if its possible to use it, via escaping, i am strongly against that as it can cause more trouble than doing good.